tests/flow/test_unwind_clause.py

```diff
@@ -0,0 +1,88 @@
+from common import *
+import re
+
+redis_graph = None
+
+class testUnwindClause(FlowTestsBase):
+    
+    def __init__(self):
+        self.env = Env(decodeResponses=True)
+        global redis_graph
+        redis_con = self.env.getConnection()
+        redis_graph = Graph(redis_con, "G")
+ 
+    def test01_unwind_null_output(self):
+        query = """UNWIND ({x:null}) AS q RETURN q.x"""
```

Comment:
`q` isn't NULL, it is a MAP object containing a key `x` with the value `NULL`
`UNWIND ({x:null}) AS q RETURN q` would have returned a valid map object.

Reply:
Ok, the test was fixed.
The unwinding null test was enabled in TCK tests.